Docs/CHANGELOG.md

```diff
@@ -1,16 +1,22 @@
 # Changelog
 
-## 0.12.0 - Unreleased
+## 0.12.0
 
 ### Vision
+- Added detection of Japanese traffic signs
 - Added property `progress` to the `VisionReplayManager`
 - Added read-only property `duration` to the `VisionReplayManager`
+- Added replay support of recorded routes
 - Changed `VisionReplayManager`'s `start`/`stop` methods functionalities to the `pause`/`resume`
```

Comment:
Oh, and this line is probably explained in the `VisionReplayManager` behavior one line below. Please remove this one.